src/Action/Api/ConfirmPaymentAction.php

```diff
@@ -32,23 +29,31 @@ public function execute($request)
         RequestNotSupportedException::assertSupports($this, $request);
 
         $model = ArrayObject::ensureArrayObject($request->getModel());
-        if (false == $model['quickpayPaymentId']) {
+        if (false === $model['quickpayPaymentId']) {
```

Comment:
Will this make it fail at some point as we saw in the factory?

Reply:
Yes probably. It is set here and does not have a default value https://github.com/Setono/payum-quickpay/blob/making-it-more-robust/src/Action/ConvertPaymentAction.php#L43